src/main/java/com/amihaiemil/docker/RtContainers.java

```diff
@@ -101,9 +101,9 @@ public Container create(
     ) throws IOException {
         final String uri;
         if(!name.isEmpty()) {
-            uri = this.baseUri.toString() + "/create?name=" + name;
+            uri = this.baseUri.toString() + "/containers/create?name=" + name;
```

Comment:
@llorllale No need to append ``/containers/`` since it is appended in ``RtDocker.containers()`` :)

Reply:
@amihaiemil ok, hadn't seen that